spec/lib/os_places_api/client_spec.rb

```diff
@@ -232,6 +232,15 @@
           },
         )
       end
+
+      it "should query OS Places API and delete the postcode if it was terminated" do
+        stub_request(:get, api_endpoint)
+          .to_return(status: 200, body: {}.to_json)
+
+        expect(Postcode.where(postcode: postcode).pluck(:postcode)).to eq([postcode])
```

Comment:
I'd make this a bit clearer with:

```suggestion
        expect(Postcode.find_by(postcode: postcode)).not_to eq(nil)
```